### PR TITLE
feed translation

### PR DESCRIPTION
**Pull request by Stefanacef**
- feed - https://wedance-stefanacef.netlify.app/feed/

### CODE REVIEW

File: pages/feed.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <THeader title="Feed">
+     <THeader :title="$t('feed.title')">
```
Comment (best-lucky1030) [inline pages/feed.vue:3]: where  this `feed.title` is used again?
Just only used here once time?

Comment (razbakov) [reply pages/feed.vue:3]: I wouldn't use `$('Feed')` so it's possible to change english translation as well, to something like `News` or `Posts` any time without changing the code.

My suggestion for i18n keys is to use pattern: `<page>.<block>.<element>.<modifier>`.

Objections?

Comment (razbakov) [reply pages/feed.vue:3]: Answer to your question: 

> where this `feed.title` is used again?

nowhere

> Just only used here once time?

yes

Comment (best-lucky1030) [reply pages/feed.vue:3]: For used once time, there is needed to add in locale file?

Comment (razbakov) [reply pages/feed.vue:3]: We have 11 languages ;)

Comment (best-lucky1030) [reply pages/feed.vue:3]: Oh, gotcha. 👍

**Review by best-lucky1030 — COMMENTED** [review]
Looks good.  ❤️

File: pages/feed.vue (hunk 1 above)
Comment (Stefanacef) [reply pages/feed.vue:3]: I read the articles and I understand the approach for naming the keys, but I don't really understand your first sentence. Can you explain to me? @razbakov

Comment (razbakov) [reply pages/feed.vue:3]: @Stefanacef you did everything right

**Review by amanuela97 — APPROVED** [review]

File: components/TEventText.vue [hunk 2]
```diff
@@ -30,7 +30,7 @@
           <div class="flex items-center gap-1">
             <TIcon name="youtube" class="w-4 h-4" />
             <p>
-              Online
+             {{$t('events.event.location')}}
```
Comment (razbakov) [inline components/TEventText.vue:33]:
```suggestion
             {{$t('events.event.online')}}
```

File: components/TEventText.vue [hunk 3]
```diff
@@ -61,10 +61,10 @@
         <div class="flex items-center gap-1">
           <TIcon name="lobby" class="w-4 h-4" />
           <div v-if="item.claimed === 'Yes'">
-            Organised by
+            {{$t('events.event.organizer')}}
```
Comment (razbakov) [inline components/TEventText.vue:64]: Please add spaces around like this, also in other places.

```suggestion
            {{ $t('events.event.organizer') }}
```

**Review by razbakov — CHANGES_REQUESTED** [review]

File: components/TLoader.vue [hunk 4]
```diff
@@ -1,3 +1,3 @@
 <template>
-  <div class="p-5 text-center">Loading...</div>
+  <div class="p-5 text-center">{{ $t('events.loading') }}</div>
```
Comment (razbakov) [inline components/TLoader.vue:2]: That component is used not only in the events page

```suggestion
  <div class="p-5 text-center">{{ $t('loading') }}</div>
```

**Review by razbakov — CHANGES_REQUESTED** [review]

File: components/TItemComments.vue [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
     <TListComments class="mt-4" :post-id="postId">
       <template v-slot:empty>
-        <div class="text-center my-8">There are no comments yet.</div>
+        <div class="text-center my-8">{{ $t('feed.post.nocomments') }}</div>
```
Comment (razbakov) [inline components/TItemComments.vue:7]: `comments.empty`

**Review by razbakov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**